src/handlers/topic1.py

```
@@ -0,0 +1,27 @@
+import logging
+from typing import Any, Dict
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)